i6_models/parts/conformer/mhsa.py

```diff
@@ -39,18 +39,28 @@ def __init__(self, cfg: ConformerMHSAV1Config):
         )
         self.dropout = cfg.dropout
 
-    def forward(self, input_tensor: torch.Tensor, sequence_mask: Optional[torch.Tensor] = None) -> torch.Tensor:
+    @staticmethod
+    def _invert_sequence_mask(sequence_mask: torch.Tensor):
```

Comment:
Maybe we want to move this function somewhere else, and call it maybe sth more expressive like `compat_logical_not`?

Reply:
we could do `util/compat.py` and then put `logical_not` in there?